jest.config.base.js

```diff
@@ -35,6 +35,7 @@ module.exports = {
         '!<rootDir>/**/test-utilities/**',
         '!<rootDir>/**/dev-scripts/**',
         '!<rootDir>/**/jump-consistent-hash.*',
+        '!<rootDir>/**/system-utils.*',
```

Comment:
why did u add this? we have test for this code. also, we usually only include test code & not product code even though if they cannot be tested

Reply:
jest does not support bigint notation when runs code coverage, fail on 1000n format